### PR TITLE
Update full-stack-developer.md

### PR DESCRIPTION
**Pull request by sparklepop**
https://strongmind.atlassian.net/browse/NW52-2863

### CODE REVIEW

File: recruit/full-stack-developer.md [hunk 1]
```diff
@@ -4,65 +4,70 @@ At StrongMind we work in terms of user stories with varying degrees of requireme
...
-The 10th frame is a bit different.  
-* If you roll a strike in the first shot of the 10th frame, you get 2 more shorts.  
-* If you roll a spare in the first two shots, you get 1 more shot.
-* If you leave the 10th frame open after two shots, the game is over and you do not get an additional shot.
-**Score:**
-The score for the 10th frame is the total number of pins knocked down in the 10th frame.
-
-More scoring details can be found [here](http://www.fryes4fun.com/Bowling/scoring.htm).
+3. Deploy your application (see below for options) 
+4. Email recruiter@strongmind.com links to your application repository and working deployed application with the subject “**Full Stack Developer Candidate - Your Name**"
```
Comment (dneighbors) [inline recruit/full-stack-developer.md:8]: make mailto: link

File: recruit/full-stack-developer.md [hunk 2]
```diff
@@ -4,65 +4,70 @@ At StrongMind we work in terms of user stories with varying degrees of requireme
...
+
+# Stories 
+
+## Toppings API 
+
+As a pizza application maker I should be able to create, read, update and delete pizza toppings.
```
Comment (dneighbors) [inline recruit/full-stack-developer.md:14]: As a pizza store owner? Also we talked about removing the API as a separate story and combining front end back end stories.

File: recruit/full-stack-developer.md [hunk 3]
```diff
@@ -4,65 +4,70 @@ At StrongMind we work in terms of user stories with varying degrees of requireme
...
+
+ 
+
+# Deployment 
+
+Your project should be deployed to a service of your choice. If you need some references to free web resources for development try https://free-for.dev  
+
+All the major cloud providers have some sort of free plan. Here are some examples: 
+
+https://azure.microsoft.com/free
```
Comment (dneighbors) [inline recruit/full-stack-developer.md:66]: these should be linkable

File: recruit/full-stack-developer.md (hunk 1 above)
Comment (sparklepop) [reply recruit/full-stack-developer.md]: updated to mailto: link

File: recruit/full-stack-developer.md (hunk 3 above)
Comment (sparklepop) [reply recruit/full-stack-developer.md]: converted to markdown link format

**Review by dneighbors — APPROVED** [review]

**Pull request merged**